std/utf.d

```diff
@@ -3532,10 +3538,209 @@ pure nothrow @nogc @safe unittest
             alias s this;
         }
 
-        auto fn = Stringish("test.d");
-        auto x = fn.byCodeUnit();
-        assert(x.front == 't');
+        auto orig = Stringish("\U0010fff8 𐁊 foo 𐂓");
+        auto bcu = orig.byCodeUnit();
+        static assert(is(typeof(bcu) == struct));
+        static assert(!is(typeof(bcu) == Stringish));
+        static assert(is(typeof(bcu) == typeof(bcu.byCodeUnit())));
+        static assert(is(ElementType!(typeof(bcu)) == immutable char));
+        assert(bcu.front == cast(char)244);
```

Comment:
Space after cast(x), here and 4 other places. It is another thing that was pointed out by the CircleCI failures, though it did not get that far because of the unittest import problem.